internal/storage/pmem/pmem.go

```diff
@@ -8,16 +8,20 @@ import (
 	"os"
 
 	"github.com/Microsoft/opengcs/internal/oc"
+	"github.com/Microsoft/opengcs/internal/storage"
+	dm "github.com/Microsoft/opengcs/internal/storage/devicemapper"
 	"github.com/pkg/errors"
 	"go.opencensus.io/trace"
 	"golang.org/x/sys/unix"
 )
 
 // Test dependencies
 var (
-	osMkdirAll  = os.MkdirAll
-	osRemoveAll = os.RemoveAll
-	unixMount   = unix.Mount
+	osMkdirAll      = os.MkdirAll
+	osRemoveAll     = os.RemoveAll
+	unixMount       = unix.Mount
+	pMemFmt         = "/dev/pmem%d"
+	linearDeviceFmt = "dm-linear-pmem%d-%d-%d"
```

Comment:
Could you move these two to a **const** block? The values in this block right now are used for mocking functions in our tests.